test/sqerl_rec_tests.erl

```diff
@@ -157,6 +157,11 @@ kitchen_test_() ->
                               "query did not return a single column",
                               [kitchen, fetch_all, []]}}},
                             sqerl_rec:scalar_fetch(kitchen, fetch_all, []))
+       end},
+      {"cinsert",
+       fun() ->
+               {K0, _Name} = make_kitchen(<<"pingpang">>),
+               {ok, 1} = sqerl_rec:cinsert(K0)
```

Comment:
`?assertEqual()` is fine, my Erlang is a bit rusty and I was just excited for it to work ;-)